### ISSUE TITLE
As a customer, I want to be able to verify my email address for the app. 

### ISSUE THREAD

**Issue by ghost** [zhoumiabc/Tofu-Kitchen-App#12]

**Comment by ghost** [comment]
Each email address can only be registered once. If a customer wants to use the same email address to register for the second time, the app will recognize it and forbid this action.

**Comment by shabbirw** [comment]
Yes, and once a customer registers, a verification email will be sent to the email that they mention. They will not be able to access their profile unless they open the verification link in their email.
